### DISCUSSION TITLE
[ FF7 ] FFNx Resolution Bug

### DISCUSSION THREAD

**Discussion by shinra358** [julianxhokaxhiu/FFNx#526] [Q&A] [answered]
720p resolution GPD Win 3 i7-1195g7

FFNX v1.14.00 stable, Resolution works correctly. A few nightlys work after that but I think ever since the widescreen mod has been put in, i may be wrong, but updating to a nightly now breaks the resolution. Stretch auto nor 16:9 no longer function and the game defaults to 4:3. The stable release, everything resolutionwise is working fine.

**Comment by julianxhokaxhiu** [comment]
Please attach your `FFNx.toml` config here that you are using. Thanks.

**Comment by shinra358** [comment]
[FFNx.toml.gz](https://github.com/julianxhokaxhiu/FFNx/files/10382640/FFNx.toml.gz)

**Comment by shinra358** [comment]
why is this switched to question?

**Comment by shinra358** [comment]
ive narrowed it down to auto resolution not working properly. Stretch to fit doesn't really do what it says but i can get reshade to fix that but auto just locks it into a hard 4:3 that reshade cant fix so i have to specify resolution

**Comment by julianxhokaxhiu** [comment, marked as answer]
There's nothing wrong with FFNx or your toml configuration file. Your expectations are probably set wrong, but whatever configuration you've chosen is meant to run the game using 640x480 by default and it will be 4:3. You'd need to tweak the configuration file to match your device resolution.
